### ISSUE TITLE
Cannot read property 'rendered' of undefined

### ISSUE THREAD

**Issue by ashutoshpw** [postlight/headless-wp-starter#142]
On yarn start, I am getting this error:

Cannot read property 'rendered' of undefined
TypeError: Cannot read property 'rendered' of undefined
    at Index.render (/root/app/headless-wp-starter/frontend/pages/index.js:62:13)
    at resolve (/root/app/headless-wp-starter/frontend/node_modules/react-dom/cjs/react-dom-server.node.development.js:2149:18)
    at ReactDOMServerRenderer.render (/root/app/headless-wp-starter/frontend/node_modules/react-dom/cjs/react-dom-server.node.development.js:2260:22)
    at ReactDOMServerRenderer.read (/root/app/headless-wp-starter/frontend/node_modules/react-dom/cjs/react-dom-server.node.development.js:2234:19)
    at renderToString (/root/app/headless-wp-starter/frontend/node_modules/react-dom/cjs/react-dom-server.node.development.js:2501:25)
    at renderPage (/root/app/headless-wp-starter/frontend/node_modules/next/dist/server/render.js:174:26)
    at Function.getInitialProps (/root/app/headless-wp-starter/frontend/node_modules/next/dist/server/document.js:83:25)
    at _callee$ (/root/app/headless-wp-starter/frontend/node_modules/next/dist/lib/utils.js:36:30)
    at tryCatch (/root/app/headless-wp-starter/frontend/node_modules/next/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/root/app/headless-wp-starter/frontend/node_modules/next/node_modules/regenerator-runtime/runtime.js:299:22)

**Comment by freddiemixell** [comment]
That means your site is receiving ‘undefined’ when looking at the WordPress REST API. Are you running the WordPress Install? Type this: localhost:PORTWHATEVER/wp-json/postlight/v1/page?slug=welcome into your browser. Is it returning an object on the page? If not you need to turn WordPress on. 

If it does return an object then check your config.js to see if the apiUrl matches the domain where your WordPress install is. 

Normally when I’m working on this repo I separate out the WordPress theme and then put it on a spare domain name that I own. Since we’re only using WordPress as cms it doesn’t really need to be on a development server. 

This way you could work very easily on a project that will look exactly identical on both sides, minus there own changes to the React frontend.

Let me know if that helps.

**Comment by ashutoshpw** [comment]
Actually I copied everything inside the wordpress/wp-content folder to my wordpress installation on the server. 

I went to

**Comment by ashutoshpw** [comment]
Actually I copied everything inside the wordpress/wp-content folder to my wordpress installation on the server.

I went to Actually I copied everything inside the wordpress/wp-content folder to my wordpress installation on the server.

I went to
MYDOMAIN/wp-json/postlight/v1/page?slug=welcome 

And it was throwing a 404 error, so I made a page on my wordpress installation with slug as welcome
And now it is working.

Thanks 👍
